www_src/pages/section-2/section-2.jsx

```diff
@@ -1,14 +1,210 @@
 var React = require('react');
 var render = require('../../lib/render.jsx');
+var Draggable = require('react-draggable');
+
+// TEMP: Just used for generating random starter layouts
+function generateLayout (width, height) {
+  var layout = [];
+
+  for (var y = 0; y < height; y++) {
+    layout.push([]);
+
+    for (var x = 0; x < width; x++) {
+      layout[y].push(Math.random() > 0.25 ? null : '/img/page-screenshot.png');
```

Comment:
don't use absolute paths, since they won't work for android (everything is inside a www/assets folder)